src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -1038,6 +1053,12 @@ internal sealed class Options : TransformInputBase
             [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "The name of the outputs", ShortName = "outputs", SortOrder = 2)]
             public string[] OutputColumns;
 
+            /// <summary>
+            /// The names of the requested key value annotations.
+            /// </summary>
+            [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "The name of the key value annotations", ShortName = "annotations", SortOrder = 3)]
+            public string[] KeyValueAnnotations;
```

Comment:
>KeyValueAnnotations [](start = 28, length = 19)

This should not be a user specified argument, it needs to be extracted from the input schema.